.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+
+## 1. Issues: Bug Reports
+One of the foundations for good software is reliability. To facilitate this reliability, our community must work together to crush bugs that arise. 
+This of course requires good information and knowledge about ongoing bugs and issues though.
+
+You can help this project by reporting a bug on our [Issues Tracker](https://github.com/TheBusyBiscuit/Slimefun4/issues).<br>
+Please adhere to the provided template and provide as much information as possible.
+For more info on how to make good and helpful bug reports, check out our article on [How to report bugs](https://github.com/TheBusyBiscuit/Slimefun4/wiki/How-to-report-bugs).
+
+If you encounter an issue which has already been reported, please don't open a new one.<br>
+It would be awesome though if you could post a comment on the existing issue which explains how you were able to reproduce this yourself.
+The more context and information we get, the easier we can fix it
+
+## 2. Pull Requests: Bug Fixes
+Bugs that have been reported need to be fixed of course.<br>
+Any open Issue on our [Issues Tracker](https://github.com/TheBusyBiscuit/Slimefun4/issues) is waiting to be fixed.
+
+This is an Open-Source project and we love Pull Requests. 
+So if you have an idea on how to approach a known issue, feel free to make a [Pull Request](https://github.com/TheBusyBiscuit/Slimefun4/pulls) which fixes this bug.
+You can also comment on the existing Issue, proposing your idea or communicating that you wanna work on this.
+
+## 3. Pull Requests: Additions/Changes
+Slimefun is an Open-Source project and anyone is allowed to make changes or add content to this plugin!
+
+Please visit our [Discord Server](https://github.com/TheBusyBiscuit/Slimefun4#discord) and share your ideas first, we hate to reject changes because the community disagrees.<br>
+So communicating your intended changes before-hand will ensure that you don't put too much work into something that might get rejected.<br>
+We also have a suggestions section in our Discord Server too, you can check out `#approved` for any suggestions the community has voted for. 
+It is a great place to start looking for ideas on what to add or change.
+
+Also consider making an addon for your additions when they get too large, check out our [Developer Guide](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Developer-Guide).
+
+## 4. Translations
+Another great way to contribute to Slimefun is by working on translations for the project.
+Slimefun's translation is available on [gitlocalize.com](https://gitlocalize.com/repo/3841).
+Just find a language you are fluent in and translate away. But make sure to submit a "Review Request" when you are done.
+One of our language moderators will review the changes and submit a Pull Request to the project for you.
+
+#### Language moderation
+Very active community translators will have the option to become a "language moderator". 
```

Comment:
I think you should capitalize the first letters of "language moderator". But this is a personal thought.

Reply:
Yeah that should probably be more consistent.